File: Java/6/testeav1/ex8.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package testeav1;
import java.util.Scanner;
/**
 *
 * @author gusta
 */
public class ex8 {
    public static void main(String[] args){
        int a,b;
        Scanner ler=new Scanner(System.in);
        System.out.println("Digite o primeiro numero");
        a=ler.nextInt();
        System.out.println("Digite o segundo numero");
        b=ler.nextInt();
        if(a>b){
            System.out.printf("%d é maior",a);
        }
        else if(b>a){
            System.out.printf("%d é maior",b);
        }
        else{
            System.out.println("São iguais");
        }
    }
}

File: C++/Estrutura/Dever1/ex8.cpp
#include <iostream>
using namespace std;
int main(){
    int vet[10], razao;
    cout<< "Digite o primeiro numero da pg";
    cin>>vet[0];
    cout<<"Digite a razao da pg";
    cin>>razao;
    for(int i=1;i<10;i++){
        vet[i]=vet[i-1]*razao;
    }
    for(int i=0;i<10;i++)
        cout<<vet[i]<<"\n";
}